tests/infra/remote.py

```diff
@@ -456,7 +457,7 @@ def __init__(
         env = {}
         self.profraw = None
         if enclave_type == "virtual":
-            self.profraw = f"{node_id}_{os.path.basename(lib_path)}.profraw"
+            self.profraw = f"{uuid.uuid4()}_{os.path.basename(lib_path)}.profraw"
```

Comment:
Can we keep the node_id here, as well as the uuid? Useful if we're ever manually looking at the individual results.

Reply:
Sure, no problem.